scripts/collect-unresolved-breaches-addresses.js

```diff
@@ -0,0 +1,79 @@
+"use strict";
+
+// TODO: Confirm db row has index
+
+const Knex = require("knex");
+const knexConfig = require("../db/knexfile");
+const knex = Knex(knexConfig);
+
+const HIBP = require("../hibp");
+
+async function checkIfBreachesExist(sha1, breaches) {
+  const breachResults = await HIBP.getBreachesForEmail(sha1, breaches, true);
+
+  if (breachResults.length >= 1) {
+    return true;
+  }
+
+  return false;
+}
+
+function getArgsValue(argument) {
+  const cliArguments = process.argv;
+
+  if (cliArguments.indexOf(argument) < 0) {
+    throw new Error(`You are missing the argument: ${argument}`);
+  }
+
+  const arguemntIndex = cliArguments.indexOf(argument);
+  const value = cliArguments[(arguemntIndex + 1)];
+
+  if (!value ) {
+    throw new Error(`No value set for ${argument}.`);
+  }
+
+  const valueNumber = parseInt(value);
+
+  if (Number.isNaN(valueNumber)) {
+    throw new Error(`The value for ${argument} is not an interger.`);
+  }
+
+  return valueNumber;
+
+}
```

Comment:
We already include **arg** module in package.json, not sure if you just want to use that vs rolling your own arg parser (as pretty as it is).
https://github.com/mozilla/blurts-server/blob/915d9fc31e8de126afc35454f8720a83c15adfcb/package.json#L11